src/contentmanager.cpp

```diff
@@ -388,6 +393,39 @@ QStringList ContentManager::getBookIds()
         }
         filter.rejectTags(tags);
     }
+
+    if (m_contentTypeFilter != "all") {
+        auto filters = m_contentTypeFilter.split(";");
+        for (auto &filter : filters) {
+            auto f = filter.split(" : ");
+            if (f[0] == gt("videos") && f[1] == gt("yes")) {
```

Comment:
Please do not do a test of the translated text.
This is somehow a golden rule. You don't know what will be the translation. Maybe in some language to English words are translated in the same text. (I doubt it could happen in this context but ... golden rule).

What we want here is probably a map, mapping a contentType ("videos", "pictures" or better, enum values) to a boolean.